templates/_layout.html

```diff
@@ -28,6 +28,16 @@
       <div class="row">
         <div class="col-12">
           &copy; 2017 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.
+          <nav>
+            <ul class="p-footer__links">
+                <li class="p-footer__item"><a class="p-footer__link" href="http://www.ubuntu.com/legal">Legal information</a></li>
+                <li class="p-footer__item"><a class="p-footer__link" href="https://github.com/ubuntudesign/snapcraft.io/issues/new">Report a bug on this site</a></li>
```

Comment:
Maybe this should link to snapcraft-flask?

But on the other hand it's meant to be part of 'snapcraft.io' site, so it would be weird to have different links here for different pages.

Reply:
It should at least be changed to `https://github.com/canonical-websites/snapcraft.io/issues/new`, as it doesn't live under `ubuntudesign` any more.

I think this link should stay as `snapcraft.io`, because I expect that this repository will ultimately live at that URL. I was thinking of doing that immediately - moving "canonical-websites/snapcraft.io" to "canonical-websites/snapcraft-homepage" or something and then creating this repository at "canonical-websites/snapcraft.io", but I decided against it as this stage because this repository is still in its infancy. As soon as it is actually released though, I think we should do those renames.